.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,7 +2,7 @@ blank_issues_enabled: true
 contact_links:
   - name: Qiskit API feedback
     url: https://github.com/Qiskit/qiskit/issues/new/choose
-    about: Open an issue in the Qiskit repository for the docs found at https://docs.quantum-computing.ibm.com/api/qiskit.
+    about: Open an issue in the Qiskit repository for the docs found at https://docs.quantum.ibm.com/api/qiskit.
```

Comment:
Bump that this change is bad in this PR. You split out your PR into two: one that solely changes the relevant URLs to use relative links, then a follow up to switch `quantum-computing` to `quantum`. This change is not ready and should be removed from this PR and only be in your follow up PR.

Reply:
Now I get it, sorry, thank You

Reply:
Hi @Eric-Arellano I wonder if you had time to check the PR? Not trying to rush anything, just double checking :), shall I create the https://docs.quantum-computing.ibm.com to https://docs.quantum.ibm.com domains one?

Thank You

Reply:
Yes - did you see my review 4 hours ago below? This PR should only be focused on relative links. We're still waiting for docs.quantum.ibm.com and that would be in a follow up PR.